templates/csharp/Skill/Skill/Adapters/DefaultAdapter.cs

```diff
@@ -53,7 +53,7 @@ public class DefaultAdapter : BotFrameworkHttpAdapter
             Use(new ShowTypingMiddleware());
             Use(new SetLocaleMiddleware(settings.DefaultLocale ?? "en-us"));
             Use(new EventDebuggerMiddleware());
-            Use(new SetSpeakMiddleware());
+            Use(new Microsoft.Bot.Builder.SetSpeakMiddleware("en-US-JennyNeural", true));
```

Comment:
```suggestion
            Use(new SetSpeakMiddleware("en-US-JennyNeural", true));
```
As we removed the one of the project, it's not necessary to keep that reference